test/lib/main.spec.js

```diff
@@ -49,13 +49,23 @@ describe('main', () => {
   });
 
   it('no argv yields usage', async () => {
-    await main([], {});
-    expect(mocks.usage.calledOnce).to.be.true;
+    try {
+      await main([], {});
+      assert.fail('Expected error to be thrown');
```

Comment:
`expect` also has `fail` and works the same as `assert.fail`. The additional require is not needed.

Reply:
Fixed.